phalcon/http/requestinterface.zep

```diff
@@ -213,10 +213,9 @@ interface RequestInterface
 	 * Checks whether request include attached files
 	 *
 	 * @param boolean onlySuccessful
-	 * @return boolean
+	 * @return int
 	 */
-	public function hasFiles(onlySuccessful = false);
-
+	public function hasFiles(boolean onlySuccessful = false);
```

Comment:
`-> bool` ?

Reply:
it actually returns the number of files https://github.com/phalcon/cphalcon/blob/master/phalcon/http/request.zep#L802

Reply:
Okay, we'll rename this method in the near future